docs/reference/units.ipynb

```diff
@@ -630,7 +634,160 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sc.scalar(4, unit='horse')"
+    "sc.scalar(4, unit='speed')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using `repr`, we can see that 'speed' is not a fundamentally new unit but simply expressed in terms of 'm' and 's':"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "repr(sc.Unit('speed'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining Scaled Units\n",
+    "\n",
+    "In the previous example, 'speed' was defined as a combination of 'm' and 's'.\n",
+    "It is also possible to define aliases of scaled units.\n",
+    "Examples of scaled units are millisecond, hour, or the previously used angstrom.\n",
+    "These units work by encoding a scale factor ('multiplier') in the unit.\n",
+    "This multiplier can be any floating point number, so for example, we can define a dog year as 52 days or 4492800 seconds:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.units.aliases['dogyear'] = sc.scalar(4492800, unit='s')"
```

Comment:
I don't want to nitpick too much, but wouldn't it be more useful to give a real example? Why not go with the lattice spacing application?

Reply:
Because everyone has some idea of what dog years are. An application specific units needs more explanation.